### PR TITLE
Replace deprecated hashdb trait names

### PR DESCRIPTION
**Pull request by keorn**

### CODE REVIEW

**Comment by keorn** [comment]
[clabot:check]

File: util/src/hashdb.rs [hunk 1]
```diff
@@ -109,7 +103,7 @@ pub trait HashDB : AsHashDB {
 	///   assert_eq!(m.get(key).unwrap(), d);
 	/// }
 	/// ```
-	fn remove(&mut self, key: &H256) { self.kill(key) }
+    fn remove(&mut self, key: &H256);
```
Comment (debris) [inline util/src/hashdb.rs]: This file is incorrectly formatted. Use tabs instead of spaces

File: util/src/hashdb.rs [hunk 2]
```diff
@@ -24,11 +24,9 @@ pub trait HashDB : AsHashDB {
 	/// Get the keys in the database together with number of underlying references.
 	fn keys(&self) -> HashMap<H256, i32>;
 
-	/// Deprecated. use `get`.
-	fn lookup(&self, key: &H256) -> Option<&[u8]>; // TODO: rename to get.
 	/// Look up a given hash into the bytes that hash to it, returning None if the
 	/// hash is not known.
-	///
+    ///
```
Comment (rphmeier) [inline util/src/hashdb.rs]: formatting still off here

File: util/src/journaldb/earlymergedb.rs [hunk 3]
```diff
@@ -290,7 +290,7 @@ impl HashDB for EarlyMergeDB {
 		ret
 	}
 
-	fn lookup(&self, key: &H256) -> Option<&[u8]> {
+	fn get(&self, key: &H256) -> Option<&[u8]> {
```
Comment (rphmeier) [inline util/src/journaldb/earlymergedb.rs]: i think this HashDB impl will fail to compile -- the deprecated methods are still required to implement the trait for backwards-compatibility reasons, and their replacements have default implementations based on those.

**Comment by gavofyork** [comment]
been waiting for this for a while.

**Pull request merged**